google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITWriteTest.java

```diff
@@ -91,6 +91,15 @@
 @Category(ParallelIntegrationTest.class)
 @RunWith(Parameterized.class)
 public class ITWriteTest {
+  private static final String RETRY_BEGIN_TRANSACTION_ON_NEW_CHANNEL_PROPERTY =
+      "com.google.cloud.spanner.retry_begin_transaction_on_new_channel";
+  private static final String originalValueForBeginTransactionOnNewChannel =
+      System.getProperty(RETRY_BEGIN_TRANSACTION_ON_NEW_CHANNEL_PROPERTY);
+
+  static {
+    System.setProperty(RETRY_BEGIN_TRANSACTION_ON_NEW_CHANNEL_PROPERTY, "true");
```

Comment:
Instead of setting this for all integration tests, shouldn't we keep the existing tests unchanged, and add maybe one new integration test with this property?